testsuite/features/step_definitions/common_steps.rb

```diff
@@ -593,15 +593,22 @@
 When(/^I enable repositories before installing Docker$/) do
   os_version, os_family = get_os_version($minion)
 
-  # Distribution Pool and Update
+  # Distribution
   repos = "os_pool_repo os_update_repo"
   puts $minion.run("zypper mr --enable #{repos}")
 
   # Tools
   repos, _code = $minion.run('zypper lr | grep "tools" | cut -d"|" -f2')
```

Comment:
That's just a matter of taste, but I would prefer to don't reuse the variable `repos` which was already defined above.

Reply:
er, a variable is for ... variable content

here it plays exactly same role, so same variable

Reply:
As I said, is just a matter of taste, but when I retrieve the outputs from a method, I prefer to use a new variable. For me is easier to handle it, in case if for instance, we wont to catch an exception there :man_shrugging:

Reply:
Maybe matter of taste yes. I find it more logical and it spares memory. I don't see why you would reuse a variable in a loop, but not in sequence.